RELEASING.md

```diff
@@ -141,6 +141,9 @@ sourcedocs generate -- -scheme WorkflowUI -workspace SampleApp.xcworkspace
 sourcedocs generate -- -scheme WorkflowTesting -workspace SampleApp.xcworkspace
 ```
 
+Note that currently sourcedocs only supports Xcode 10, if you run it with Xcode 11 you might see
```

Comment:
Is this a `Sourcedocs` issue or is this a configuration issue on our side?

Reply:
Seems like a Sourcedocs issue. Filed a bug: https://github.com/eneko/SourceDocs/issues/30